ocfweb/api/staff_hours.py

```diff
@@ -0,0 +1,13 @@
+from json import JSONEncoder
+
+from django.http import JsonResponse
+
+from ocfweb.main.staff_hours import get_staff_hours as real_get_staff_hours
+
+
+def get_staff_hours(request):
+    return JsonResponse(
+        list(map(lambda item: item._asdict(), real_get_staff_hours())),
+        encoder=JSONEncoder,
+        safe=False,
```

Comment:
These two lines probably are unnecessary.

Reply:
Yes, I think it would be better to use the default Django encoder. Is there a reason you want to use the `json` encoder?

Reply:
Nope, this was my mistake